Assets/Scripts/Wood.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Wood : MonoBehaviour
+{
+    public float size;
```

Comment:
I am really not so sure about this, a script just to contain a variable is always unfortunate. Maybe we can add a fuel size variable to the item association script? Then anything can burn instead of only wooden logs
(+ I think we will be using an 'item' tag, which means we can check faster if the thing which is colliding with the fire is something worth evaluating, instead of trying to do a getComponent every frame)

Reply:
So I can add the attribute fuelSize to the Item class?